app-main/public/css/style.css

```diff
@@ -472,15 +468,46 @@ span.lessons-count {
 }
 
 .create-lesson-input {
+  outline: none;
   margin-bottom: 20px;
   padding: 12px;
   border: 1px solid #ccc;
 }
 
-input.create-lesson-input {
+.create-lesson-input-invalid {
+  outline: none;
+  margin-bottom: 20px;
+  padding: 12px;
+  border: 1px solid red;
+}
+
+input.create-lesson-input,
+input.create-lesson-input-invalid {
   flex: 1;
 }
 
+span.create-lesson-input-tooltip::after {
```

Comment:
I wonder if maybe we're better off sticking with the native HTML form validation? i.e. by using the `required` attribute. I think this code is still useful though and we shouldn't remove it. We will need tooltips at some point (we probably need them for the lesson tags).

Because we have some positioning issues on smaller screen sizes I wonder if we just leave the native HTML validation tooltip to do its job. What do you think?

Reply:
Sure I think that's the best option. I'll remove everything but the CSS code in regards to the custom tooltip and fix the issue with the required attribute instead.